### PR TITLE
Security gear armor buff

### PR DESCRIPTION
**Pull request by starwolfy**
## About the PR

Improved armor protection values for the standard security helmet and the standard security armored vest. Helmets have an additional +5% armor for blunt, slash and piercing. Security vests an additional +5% for blunt and slash and an additional +3% for piercing protection.
## Why / Balance

Make 1v1s in edges of the stations more winnable for security officers, including unanticipated attacks from crew during revs for more effective and rewarding security role gameplay. Survive just a hit or two more. Make security great again.

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Buffed standard Security armor.

## Summary by CodeRabbit

- **Gameplay Changes**
	- Adjusted armor effectiveness for helmets, resulting in increased damage taken from Blunt, Slash, Piercing, and Heat attacks.
	- Modified armor coefficients for outer clothing, leading to higher damage susceptibility from Blunt and Slash attacks, and a slight increase for Piercing attacks.

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Head/helmets.yml [hunk 1]
```diff
@@ -15,10 +15,10 @@
...
     modifiers:
       coefficients:
-        Blunt: 0.9
-        Slash: 0.9
-        Piercing: 0.9
-        Heat: 0.9
+        Blunt: 0.85
+        Slash: 0.85
+        Piercing: 0.85
+        Heat: 0.85
```
Comment (whateverusername0) [inline Resources/Prototypes/Entities/Clothing/Head/helmets.yml:21]: sec armor is not supposed to be heat resistant change it back :godo:

**Review by whateverusername0 — CHANGES_REQUESTED** [review]

**Comment by JoeHammad1844** [comment]
resistances that dont end in 5 or 0 just make me feel weird

**Comment by JoeHammad1844** [comment]
sec powercreep by the way

**Comment by Piras314** [comment]
add changelog

File: Resources/Prototypes/Entities/Clothing/Head/helmets.yml (hunk 1 above)
Comment (Piras314) [reply Resources/Prototypes/Entities/Clothing/Head/helmets.yml:21]: it had heat resistance before it seems

**Comment by Piras314** [comment]
@starwolfy add a changelog

File: Resources/Prototypes/Entities/Clothing/Head/helmets.yml (hunk 1 above)
Comment (whateverusername0) [reply Resources/Prototypes/Entities/Clothing/Head/helmets.yml:21]: i meant to change i back to 0.9

**Review by whateverusername0 — APPROVED** [review]

**Comment by Piras314** [comment]
Okay I'll add a changelog I guess

**Pull request merged**